src/styles/calendar.scss

```diff
@@ -0,0 +1,50 @@
+$size: 100px;
+$month-length: 31;
+$sun: 606px;
+
+.calendar {
+  gap: 1px;
+  display: flex;
+  box-sizing: border-box;
+  width: 726px;
+  height: fit-content;
+  flex-wrap: wrap;
+
+  position: absolute;
+  inset: 0;
+  margin: auto;
+
+  padding: 10px;
+  background: #fff;
+  font-size: 30px;
+
+  &-day {
+    display: flex;
+    box-sizing: border-box;
+    width: $size;
+    height: $size;
+    align-items: center;
+    justify-content: center;
+    background-color: #eee;
+    border: 1px solid #000;
+
+    @each $shift in $sun {
+      &:nth-child(1) {
+        margin-left: $shift;
+      }
```

Comment:
In this loop, you are applying the `margin-left` property to the first child of `.calendar-day` for every value in `$sun`. This will result in only the last value in `$sun` being applied, since CSS rules are applied in order, and later rules override earlier ones. This is not the intended behavior.